rebar.config

```diff
@@ -1,24 +1,35 @@
 {deps, [
-    {lager, {git, "git://github.com/basho/lager", {tag, "3.2.1"}}},
+    {lager, "3.2.1"},
     %%{riak_core, {git, "https://github.com/project-fifo/riak_core", {branch, "fifo-0.7.1"}}},
+    {riak_core, "2.2.6", {pkg, riak_core_ng}},
     {riak_dt,  {git, "git://github.com/aletomsic/riak_dt", {tag, "type_check_antidote"}}},
     %% TODO: riak_pb branch "antidote_crdt"
     {riak_pb, {git, "git://github.com/syncfree/riak_pb", {branch, "antidote_crdt_rebar3"}}},
     {riak_api, {git, "git://github.com/basho/riak_api", {tag, "2.0.2"}}},
     {erlzmq, {git, "git://github.com/tcrain/erlzmq2", {branch, "master"}}},
     %% antidote_pb is client interface. Needed only for riak_tests.
-    {antidote_pb, {git, "git://github.com/syncfree/antidote_pb", {tag, "new_interface"}}},
-    {antidote_crdt, ".*", {git, "git://github.com/syncfree/antidote_crdt", {tag, "0.0.1"}}},
+    {antidote_pb, {git, "git://github.com/syncfree/antidote_pb", {branch, "new_interface_erl19"}}},
+    {antidote_crdt, ".*", {git, "git://github.com/syncfree/antidote_crdt", {tag, "0.0.2"}}},
     {antidote_utils, {git, "git://github.com/syncfree/antidote_utils", {tag, "0.0.1"}}}
 ]}.
 
 {erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform},{i,"include"}]}.
 {cover_enabled, true}.
 {eunit_opts, [verbose, {report, {eunit_surefire, [{dir,"logs/"}]}}]}.
+{dialyzer, [{warnings, [
+  %unmatched_returns,
```

Comment:
I had to add the `no_undefined_callbacks` option, because there is some dialyzer error in riak_dt, which leads to an error "Callback info about the riak_dt behaviour is not available".

When I added that option, I looked at the other options in the dialyzer manual and added some (commented) options which might be useful to add later. However, I did not want to mix Erlang19 changes with other typing stuff.